com.unity.ml-agents/package.json

```diff
@@ -5,7 +5,7 @@
   "unity": "2023.2",
```

Comment:
Do you need to bump the version? And update the changelog?

Reply:
Changelog yes, version should be ok I think. Did 2023.3 come out already?